packages/buttons/src/_selectors.css

```diff
@@ -1,22 +1,15 @@
-/* stylelint-disable at-rule-empty-line-before */
+@custom-selector :--btn-hover
+  :hover,
+  .is-hovered;
 
-@custom-selector :--btn-primary
-  .c-btn--primary,
-  .is-selected;
-
-@custom-selector :--btn-hovered
-  .is-hovered.is-hovered,
-  :hover:hover;
-
-@custom-selector :--btn-focused
+@custom-selector :--btn-focus
+  :focus-visible,
```

Comment:
With [less than 5% of browsers supporting this](https://caniuse.com/#search=focus-visible) and the [proposal still being a working draft](https://drafts.csswg.org/selectors-4/#the-focus-visible-pseudo) I'm not sure it's the right time to add this. Requiring consumers to include an additional polyfill, even if it is official, seems at odds with providing native CSS.

Reply:
This is a more mainstream option than the previous `.is-focused`. Now both are supported and folks don't need to roll their own custom JS to achieve the right focus/click behavior.